Resolve warnings

dkp1903 opened the pull request.

pranavpandey1998official commented on package-lock.json:
```diff
@@ -1624,14 +1624,6 @@
         "react-transition-group": "^4.3.0"
       }
     },
-    "@material-ui/icons": {
-      "version": "4.9.1",
-      "resolved": "https://registry.npmjs.org/@material-ui/icons/-/icons-4.9.1.tgz",
-      "integrity": "sha512-GBitL3oBWO0hzBhvA9KxqcowRUsA0qzwKkURyC8nppnC3fw54KPKZ+d4V1Eeg/UnDRSzDaI9nGCdel/eh9AQMg==",
-      "requires": {
-        "@babel/runtime": "^7.4.4"
-      }
-    },
```
why you delete this bro

pranavpandey1998official commented on .gitignore:
```diff
@@ -8,6 +8,7 @@
 # testing
 /coverage
 
+/.vscode
```
why

pranavpandey1998official commented on .vscode/settings.json:
```diff
@@ -0,0 +1,3 @@
+{
+    "deno.enable": false
+}
```
why

dkp1903 commented:
Sorry bro, i had no idea package.json me koi change hua. Bcz I had pulled
your latest changes, and uske baad npm.i kuchh nahi kiya.

Vo vs code wala happened bcz mere VS code me Deno extension enabled tha, to
usko disable karne ke liye aisa hua tha. But then, i just removed the
extension, and added the vscode folder to ignore

On Thu, Jun 25, 2020, 23:44 pranavpandey1998official <
notifications@github.com> wrote:

> *@pranavpandey1998official* requested changes on this pull request.
> ------------------------------
>
> In .gitignore
> <https://github.com/pranavpandey1998official/webClientSEN/pull/37#discussion_r445746668>
> :
>
> > @@ -8,6 +8,7 @@
>  # testing
>  /coverage
>
> +/.vscode
>
> why
> ------------------------------
>
> In .vscode/settings.json
> <https://github.com/pranavpandey1998official/webClientSEN/pull/37#discussion_r445746839>
> :
>
> > +{
> +    "deno.enable": false
> +}
>
> why
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/pranavpandey1998official/webClientSEN/pull/37#pullrequestreview-437741549>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJJ3Y6PWUV657XNZZG5Y3TTRYOHY7ANCNFSM4OIB4DHQ>
> .
>
